cmake/modules/Libbacktrace.cmake

```diff
@@ -0,0 +1,18 @@
+include(ExternalProject)
+
+ExternalProject_Add(project_libbacktrace
+  PREFIX libbacktrace
+  SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/libbacktrace"
+  CONFIGURE_COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/libbacktrace/configure" "--prefix=${CMAKE_CURRENT_BINARY_DIR}/libbacktrace"
+  INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/libbacktrace"
+  BUILD_COMMAND make
+  INSTALL_COMMAND make install
```

Comment:
Question: How cmake generate for Windows with this command?

Reply:
Good question, I still need to look at windows support.

Reply:
This does not work on windows. libbacktrace only has support for building using configure which requires bash and bash is only available on windows via projects such as msys or cygwin which seems like too large a prerequisite for windows.

In the short term I have disabled support for libbacktrace on windows which should still give us support on other platforms. For windows we should make a cmake build script for libbacktrace which is fairly easy. We can patch that in until it can be upstreamed to libbacktrace.

Reply:
Thank you for the help @rkimball!